vagrant/provisioning/roles/arkcase-file-utils/tasks/main.yml

```diff
@@ -84,10 +84,16 @@
     dest: /tmp/
   when: not libreoffice_bin_exists.stat.exists
 
+- name: Ensure that libreoffice7.3-freedesktop-menus-7.3.5-2.noarch.rpm is removed
```

Comment:
here you should encode the `libreoffice_version` fact, or else remove any RPM with `freedesktop` in the filename.  As it is now, if someone sets the `libreoffice_version` to 7.3.6, and it also has a freedesktop-menus package, it will break.